src/Request.hs

```diff
@@ -103,18 +105,18 @@ mkValidator contractInfo@ContractInfo{..} dat lenderTn ctx = validate
     findDatumHash' datum info = findDatumHash (Datum $ toBuiltinData datum) info
 
     expectedNewDatum :: POSIXTime -> Collateral.CollateralDatum
-    expectedNewDatum ld = Collateral.CollateralDatum { 
+    expectedNewDatum ld = Collateral.CollateralDatum {
         Collateral.borrowersNftTn        = borrowersNftTn dat
-      , Collateral.borrowersPkh          = borrowersPkh dat
+      , Collateral.borrowersAddress      = borrowersAddress dat
```

Comment:
Is this field necessary in the following logic? Can we remove it instead?

Reply:
it could be removed as it used only in `Request.hs` validator but that would complicate backend logic a bit so I think it would be better to leave it as it is for now

Reply:
ok 👍